exporters/common/src/main/java/io/opentelemetry/exporter/internal/grpc/GrpcExporterBuilder.java

```diff
@@ -108,6 +110,13 @@ public GrpcExporterBuilder<T> configureKeyManager(byte[] privateKeyPem, byte[] c
     return this;
   }
 
+  public GrpcExporterBuilder<T> setSslSocketFactory(
+      SSLSocketFactory sslSocketFactory, X509TrustManager trustManager) {
+    tlsConfigHelper.setSslSocketFactory(sslSocketFactory);
+    tlsConfigHelper.setTrustManager(trustManager);
+    return this;
+  }
```

Comment:
We should add this method to `OkHttpExporterBuilder` as well in place of `OkHttpExporterBuilder#setSslSocketFactory` and `OkHttpExporterBuilder#setTrustManager`.

Reply:
Thanks, yeah, I missed those. Will fix that up.